src/OpenRoad.tcl

```diff
@@ -111,14 +111,19 @@ proc write_def { args } {
   ord::write_def_cmd $filename $version
 }
 
-sta::define_cmd_args "write_abstract_lef" {filename}
+sta::define_cmd_args "write_abstract_lef" {[-bloat amount] filename}
 
 proc write_abstract_lef { args } {
-  sta::parse_key_args "write_abstract_lef" args keys {} flags {}
+  sta::parse_key_args "write_abstract_lef" args keys {-bloat} flags {}
+
+  set bloat 2
```

Comment:
I disagree on this point.  The current behavior is not equivalent to zero and it isn't a good default as it will produce overly detailed abstract (really no abstraction at all).
@arlpetergadfort would you comment further on your thoughts?

Reply:
@maliberty i was wrong. I agree with @rovinski that the 0 bloat behavior should revert to the current implementation. That is what I meant.